tests/controls/virtlistctrltest.cpp

```diff
@@ -105,4 +109,41 @@ void VirtListCtrlTestCase::UpdateSelection()
     CPPUNIT_ASSERT_EQUAL( 1, m_list->GetSelectedItemCount() );
 }
 
+void VirtListCtrlTestCase::DeselectedEvent()
+{
+#if wxUSE_UIACTIONSIMULATOR
+    m_list->SetItemCount(1);
```

Comment:
Adding `m_list->AppendColumn("Col0");` makes the test pass.

Reply:
Ah, that's right! Thanks! A silly but fundamental error.